hangman/hangman.py

```diff
@@ -3,12 +3,25 @@
 random_word = str(random.choice(my_list).upper())
 print(random_word)
 
-word = input("Kindly type one letter only: ").upper()
-index = 0
-# print(word)
-for i in random_word:
-    if word == i:
-        print("YES in {}".format(index))
-    index += 1
-if word not in random_word:
-    print("NO")
\ No newline at end of file
+dic_list = {}
+for letters in random_word:
+    dic_list[letters] = False
+print(dic_list)
+
+word = ""
+count = 0
+while word != dic_list and word != "!":
+    word = input("Kindly type one letter only: ").upper()
+    index = 0
+    count += 1
+    for i in dic_list:
```

Comment:
loop through `random_word` instead of `dic_list` here. Characters in `dic_list` only appear once even if they appear multiple times in the word